harvest/algo.py

```diff
@@ -375,7 +375,7 @@ def get_price(self, symbol: str=None) -> float:
         if symbol == None:
             symbol = self.watch[0]
         if len(symbol) <= 6:
-            return self.trader.storage.load(symbol, self.trader.interval)['close']
+            return self.trader.storage.load(symbol, self.trader.interval)[symbol]['close'][-1]
```

Comment:
I feel like having to index a dictionary for the `symbol` when you specify the symbol in the function is bad code. Storage should keep the code as an single level index.

Reply:
I think I'll make this an issue around v0.3 or something, for now it'll do 😅